.slim-lint.yml

```diff
@@ -26,10 +26,13 @@ linters:
       - Layout/HashAlignment
       - Layout/IndentationWidth
       - Layout/TrailingEmptyLines
+      - Layout/LineLength
```

Comment:
I'm ok excluding this on the slim linting, since it's hard to keep the same line length formatting when you're dealing with HTML & ruby combined.